content/support-training.html

```diff
@@ -0,0 +1,173 @@
+            <img src="../assets/commercial-support-logos/credativ.png" alt="Credativ">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://grafana.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/grafana-labs.svg" alt="Grafana Labs">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://juliusv.com/" class="btn btn-training btn-default btn-large" target="_blank">
+            Julis Volz <br/> 
+            (Independent Contractor)
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://tina.tincho.org" class="btn btn-training btn-default btn-large" target="_blank">
+            Martina Ferrari <br/> 
+            (Independent Contractor)
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.metricfire.com/hosted-prometheus" class="btn btn-training btn-default darkgray-bg btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/metricfire.png" alt="MetricFire">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/robust-perception.png" alt="Robust Perception">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          
+          <a href="https://www.robustperception.io/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/sysdig.svg" alt="Sysdig">
+          </a>
+      </div>
+      
+      <div class="col-md-3 col-sm-6 doc-content tc-col">
+          <a href="https://www.weave.works/features/prometheus-monitoring/" class="btn btn-training btn-default btn-large" target="_blank">
+            <img src="../assets/commercial-support-logos/weaveworks.svg" alt="Weaveworks">
+          </a>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="container">
+      <div class="col-md-12 doc-content">
+        <a name="add"></a><h2>Add a resource</h2>
+        <p>
+          Anyone who has developed a Prometheus training program or offers related services
+          can add themselves to this page if they meet the following requirements.
+        </p>
+        </div>
+        
+        <div class="col-md-6 doc-content">
+          <!-- NOTE: WIP -->
+          <b>For courses and training programs:</b>
+          <ul>
+            <li>The course must have a publicly available syllabus or outline on the page linked.</li>
```

Comment:
consistency nit: first bullet point has a period, the others don't (either way is fine as long as it's consistent)